core/src/main/java/com/opensymphony/xwork2/util/AnnotationUtils.java

```diff
@@ -151,4 +152,33 @@ public static String resolvePropertyName(Method method) {
 
         return ann;
     }
+
+    /**
+     * Returns the annotation on the given class or the package of the class. This searches up the
+     * class hierarchy and the package hierarchy for the closest match.
+     *
+     * @param <T>             class type
+     * @param clazz           The class to search for the annotation.
+     * @param annotationClass The Class of the annotation.
+     * @return The annotation or null.
```

Comment:
never return 'null'

Reply:
never returns `null`

Reply:
Sorry, I forgot to update the JavaDoc :)